# msg de vetores

float64[] data